scripts/thaliPrerequisite.js

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
```

Comment:
Have you tested at all what happens if one of the commands below, the script fails and the user tries again? For example, what if GitHub is down? Can they run again or is the state now a hopeless mess and hey need to delete their project and start over?